### PR TITLE
gui/settings dialog: Add option for disable video player.

### PR DESCRIPTION
**Pull request by Zangetsu38**
create this pr for some game can get playable/ingame with bypass video player support not totally working.

- get akiba/diva f and more to ingame

### CODE REVIEW

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -246,6 +246,9 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Checkbox("Disable experimental ngs support", &host.cfg.disable_ngs);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Disable experimental support for advanced audio library ngs");
+        ImGui::Checkbox("Enable video playing support", &host.cfg.video_playing);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Uncheck the box to disable video player.\nOn some game, disable it request for more progress.");
```
Comment (pent0) [inline vita3k/gui/src/settings_dialog.cpp:251]:
```suggestion
            ImGui::SetTooltip("Uncheck the box to disable video player.\nOn some game, disable it is required for more progress.");
```

File: vita3k/gui/src/settings_dialog.cpp [hunk 2]
```diff
@@ -416,10 +419,20 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
+    if (host.cfg.overwrite_config) {
+        ImGui::Spacing();
+        ImGui::Separator();
+        ImGui::Spacing();
+        static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
+        ImGui::SetCursorPosX((ImGui::GetWindowSize().x / 2.f) - (BUTTON_SIZE.x / 2.f));
+        if (ImGui::Button("Save", BUTTON_SIZE))
+            config::serialize_config(host.cfg, host.cfg.config_path);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Click on Save is request keep change after reboot.");
```
Comment (pent0) [inline vita3k/gui/src/settings_dialog.cpp:433]:
```suggestion
            ImGui::SetTooltip("Click on Save is required to keep changes after reboot.");
```

**Review by pent0 — CHANGES_REQUESTED** [review]

**Review by pent0 — DISMISSED** [review]

**Review by pent0 — APPROVED** [review]

File: vita3k/gui/src/settings_dialog.cpp [hunk 3]
```diff
@@ -246,6 +246,9 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::Checkbox("Disable experimental ngs support", &host.cfg.disable_ngs);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Disable experimental support for advanced audio library ngs");
+        ImGui::Checkbox("Enable video playing support", &host.cfg.video_playing);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Uncheck the box to disable video player.\nOn some game, disable it is required for more progress.");
```
Comment (KorewaWatchful) [inline vita3k/gui/src/settings_dialog.cpp:251]:
```suggestion
            ImGui::SetTooltip("Uncheck the box to disable the video player.\nOn some games, disabling it is required for farther progress.");
```

File: vita3k/gui/src/settings_dialog.cpp [hunk 4]
```diff
@@ -416,10 +419,20 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
...
+    if (host.cfg.overwrite_config) {
+        ImGui::Spacing();
+        ImGui::Separator();
+        ImGui::Spacing();
+        static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
+        ImGui::SetCursorPosX((ImGui::GetWindowSize().x / 2.f) - (BUTTON_SIZE.x / 2.f));
+        if (ImGui::Button("Save", BUTTON_SIZE))
+            config::serialize_config(host.cfg, host.cfg.config_path);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Click on Save is required to keep changes after reboot.");
```
Comment (KorewaWatchful) [inline vita3k/gui/src/settings_dialog.cpp:433]:
```suggestion
            ImGui::SetTooltip("Click Save to keep changes after reboot.");
```

**Review by KorewaWatchful — CHANGES_REQUESTED** [review]

File: vita3k/gui/src/settings_dialog.cpp (hunk 4 above)
Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:433]: Hmm bent have already review and fix it 😅

Comment (KorewaWatchful) [reply vita3k/gui/src/settings_dialog.cpp:433]: Alright, I'll do a separate PR for all the grammar stuff lol.

**Pull request merged**